src/main/scala/com/gu/acquisitionFirehoseTransformer/AcquisitionToJson.scala

```diff
@@ -0,0 +1,30 @@
+package com.gu.acquisitionFirehoseTransformer
+
+import java.text.SimpleDateFormat
+import java.util.Date
+
+import io.circe.Json
+import io.circe.generic.auto._
+import io.circe.syntax._
+import ophan.thrift.event.Acquisition
+
+object AcquisitionToJson {
+  private val formatter = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
+
+  private case class AcquisitionOutput(
+    paymentfrequency: String,
```

Comment:
in the current acquisitions tables in the data lake, fields are underscore-separated. it would be helpful if these were too